Conceptpower+Spring/src/test/java/edu/asu/conceptpower/core/ConceptManagerTest.java

```diff
@@ -147,14 +149,8 @@ public void testAddConceptListEntryToWordnet() throws DictionaryDoesNotExistExce
 		newConcept.setPos("noun");
 		newConcept.setWord("test new");
 		newConcept.setWordnetId("WID-1");
-		
-		String id = null;
-		try {
-			id = managerToTest.addConceptListEntry(newConcept);
-		} catch (DictionaryModifyException e) {
-			// test passes
-		}
-		
-		Assert.assertNull(id);		
+		expectedEx.expect(DictionaryModifyException.class);
+		String id = managerToTest.addConceptListEntry(newConcept);
+		Assert.assertNull(id);
```

Comment:
You're still asserting that the id is null, which it shouldn't because an exception is thrown so there is no return value, right?